src/graphics/sdl/sdl_texture.cpp

```
@@ -62,6 +62,17 @@ namespace moth_graphics::graphics::sdl {
         SDL_SetRenderTarget(m_renderer, prevTarget);
     }
 
+    void Texture::UpdatePixels(IntRect const& destRect, uint8_t const* pixels) {
+        if (pixels == nullptr) {
+            return;
+        }
+        SDL_Rect const sdlRect = ToSDL(destRect);
```

Clang (14.0.6)

[error] variable 'sdlRect' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)